src/app/pages/common/ix-tables/components/ix-table-paginator/ix-table-paginator.component.ts

```diff
@@ -41,7 +41,7 @@ export class IxTablePaginatorComponent<T> extends MatPaginator {
   }
 
   get total(): number {
-    return this.dataSource.data.length || 0;
+    return this.dataSource?.data.length || 0;
```

Comment:
If data is undefined, wouldn't this still give an error of: "Cannot read length of undefined"?